features/performance/manifests/01-tuned-rt.yaml

```diff
@@ -23,10 +23,15 @@ spec:
       vm.stat_interval = 10
       kernel.timer_migration = 0
 
+      [sysfs]
+      /sys/bus/workqueue/devices/writeback/cpumask = ${not_isolated_cpumask}
+      #/sys/devices/virtual/workqueue/cpumask = ${not_isolated_cpumask}
+      /sys/devices/system/machinecheck/machinecheck*/ignore_ce = 1   
+
       [scheduler]
       isolated_cores=${isolated_cores}
     name: openshift-realtime-node
-  recommend:
+  recommend:  
```

Comment:
does probably not break anything, but looks like we have 2 new trailing spaces here